lib/ts/controllers/s-popover.ts

```diff
@@ -78,6 +78,11 @@ namespace Stacks {
             this.popper.enableEventListeners();
 
             this.popoverElement!.classList.add("is-visible");
+
+            // If the popover has undergone a content change while hidden, it may need to update its position slightly.
+            // Scheduling an update will cause the change to happen at the next animation frame.
+            this.popper.scheduleUpdate();
```

Comment:
We're calling `popper.update` above, which *should* be taking care of the positioning. It's possible that it doesn't position right until being made visible though... Needs a second look. I'd either remove this line and move the `.update` down to here, or remove the `.update` above so we don't double up on the positioning calls.

Reply:
I was running into the issue with the following markup, possibly because of line wrapping.

When the popover would first appear, it would be a smidge too high, then when I moved the popover would move to the correct position.  Just calling `scheduleUpdate()` seems to be sufficient without any visual movement of the element.

```
<button class="grid--cell s-btn s-btn__unset" data-controller="s-tooltip" data-s-tooltip-placement="right" title="Downvote this answer if you find it unclear or not useful"><svg aria-hidden="true" class="svg-icon m0 iconArrowDownLg" width="36" height="36" viewBox="0 0 36 36"><path d="M2 10h32L18 26 2 10z"/></svg></button>
```